.github/workflows/hrm-production-release.yml

```diff
@@ -0,0 +1,37 @@
+# Workflow to create a new production release from a given qa tag
+
+name: Create a production release
+
+# Controls when the action will run.
+on:
+  workflow_dispatch:
+    inputs:
+      title:
+        description: Release title - The title that will be given to this pre-release.
+        required: true
+
+jobs:
+  generate-release:
+    runs-on: ubuntu-latest
+
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+
+      - name: Create production release
+        uses: techmatters/flex-plugins/.github/actions/generate-production-release@gian_CHI-1264
+        # uses: techmatters/flex-plugins/.github/actions/generate-production-release@master
```

Comment:
TODO: pin to master after https://github.com/techmatters/flex-plugins/pull/1147 is merged, prior to merging this one.